src/app/shared/clarin-item-box-view/clarin-item-box-view.component.ts

```diff
@@ -200,11 +200,11 @@ export class ClarinItemBoxViewComponent implements OnInit {
         getFirstCompletedRemoteData(),
         switchMap((clList: RemoteData<PaginatedList<ClarinLicense>>) => clList?.payload?.page))
       .subscribe(clarinLicense => {
+        let iconsList = [];
         clarinLicense.extendedClarinLicenseLabels.forEach(extendedCll => {
-          this.licenseLabelIcons.push(extendedCll?.icon);
+          iconsList.push(extendedCll?.icon);
         });
-        // For now show only extended CLL icons
-        // this.licenseLabelIcons.push(clarinLicense?.clarinLicenseLabel?.icon);
+        this.licenseLabelIcons.next(iconsList);
```

Comment:
what was wrong with the previous approach?

Reply:
Because there should be visible only icons of the extended license labels in the item view. Previously the non extended license labels could be seen also.